### ISSUE TITLE
total_hosts in Model is actually not total hosts, it is "all different individuals"

### ISSUE THREAD

**Issue by wenzeslaus** [ncsu-landscape-dynamics/pops-core#91]
I realized there was a reason I never renamed `total_plants` to `total_hosts`. Even in simple SI model without pest resistance, `S + I != total_plants` because the model includes a dilution effect where number of all (not only host) plants in the cell influences the establishment:

```
double probability_of_establishment =
                                (double)(susceptible(row, col)) / total_hosts(row, col);
```

https://github.com/ncsu-landscape-dynamics/PoPS/blob/master/include/pops/simulation.hpp#L419

This is in the `disperse` function.

On the other hand, the newer `movement` function assumes `S + I == total_hosts`. I don't think the limitation/assumption in `movement` necessitates that the whole model needs to require the same condition as long as it is documented that the `movement` can only be used when the condition is met.

**Comment by ChrisJones687** [comment]
Would a better parameter name be total_populations? This would get past the animal and plant issue while being more correct than total hosts and then we just define in the documentation that total_populations is the cumulative hosts and non-hosts in a cell and must be >= S + E + I.

Agreed that the rest of the model doesn't need to and shouldn't mimic that assumption in the movements module. We should also make this assumption more flexible when we address #87.

**Comment by wenzeslaus** [comment]
Fixed by #98.
